### ISSUE TITLE
fasttext

### ISSUE THREAD

**Issue by daleangus** [castorini/hedwig#52]
is this the Facebook FastText model?

**Comment by achyudh** [comment]
Yeah https://github.com/castorini/hedwig/pull/38 adapts the model from https://arxiv.org/pdf/1607.01759.pdf for document classification, though you might have to dig into the implementation to see if there are differences between our model and Facebook's implementation
